libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -322,15 +322,18 @@ protected void showError(Exception exception) {
     public void loadLoginPage() {
         if (TextUtils.isEmpty(loginOptions.getJwt())) {
             loginOptions.setLoginUrl(getLoginUrl());
-            doLoadPage(false);
```

Comment:
We don't need to pass a jwtFlow boolean, we can figure that out by seeing if there is a jwt token in the LoginOptions.